examples/tutorials/Ordinary_Differential_Equation_Solving_using_deepchem.ipynb

```diff
@@ -109,7 +109,7 @@
   },
```

Comment:
<p>Use US spelling "minimizer" as default to match style elsewhere.</p><p> </p><p>Also add some scientific discussion of how the parameter estimation works</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>